apps/changelog/classes/log_entry.py

```diff
@@ -0,0 +1,147 @@
+# Python Standard Library Imports
+from collections import namedtuple
+
+# Third Party (PyPI) Imports
+import dateparser
+import emoji
+
+# HTK Imports
+from htk.apps.changelog.classes.release_version import ReleaseVersion
+from htk.apps.changelog.constants.general import SEPERATOR
+from htk.apps.changelog.constants.regexes import (
+    GITHUB_ISSUE_REGEX,
+    RELEASE_TAG_REGEXES,
+)
+
+
+class LogEntry(
+    namedtuple(
+        'LogEntry',
+        'origin_url,commit_hash,date_iso,author,refs_raw,subject',
+    )
+):
```

Comment:
# Reviewer Note
Python 2 compatible code to achieve `@dataclass` decorator in Python 3